### PR TITLE
feat: Warning about MDW out of sync 

### LINKED ISSUE #605: Warning about MDW out of sync
### Please describe the problem that your request should resolve.
currently testnet mdw is not synced (already pinged the team). Could we show a warning to the user in aescan if thats the case to avoid confusion?

https://testnet.aeternity.io/mdw/status e.g. if "mdw_synced": false and mdw/node height have a difference > 5?

### PR DESCRIPTION
**Pull request by janmichek**
## Description
resolves #605

## Demo
image

image

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] add syncing warning UI

### CODE REVIEW

**Review by Liubov-crypto — CHANGES_REQUESTED** [review]
I got periodically 404 errors in FF, Chrome and Safari browsers. 

Also I got these issues: 

https://github.com/aeternity/aescan/assets/69896204/1ee87688-3dba-4733-8425-b0ba2337ac91

errr1
cons

**Comment by janmichek** [comment]
> I got periodically 404 errors in FF, Chrome and Safari browsers.
> 
> Also I got these issues:
> 2024-03-18.5.19.30.mov
> 
> errr1 cons

I think it was caused by the deployment low resources. Can you retry now? However I am not sure how to test this, as wee need to have MDW in syncing state.

**Comment by Liubov-crypto** [comment]
It was not working for me in the testnet. 
In mainnet I didn't get such issues as before. 
Probably you have a right, I have to catch somehow middleware out of sync to test it properly. I tried to change my network settings, but it seems to be different and it requires another approach.

File: src/components/TheHeader.vue [hunk 1]
```diff
@@ -37,13 +37,23 @@
           'header__network-select',
           { 'header__network-select--open': isNavigationOpen }]"/>
     </div>
+    <div
+      v-if="isSyncing"
+      class="header__warning">
+      The blockchain status is currently being synced. The data accuracy might be affected. Please check again
```
Comment (michele-franchi) [inline src/components/TheHeader.vue:43]:
```suggestion
      Some services are currently being synced and data accuracy might be affected. Please check again
```

**Review by michele-franchi — CHANGES_REQUESTED** [review]
I simulated it locally and looks good to me but I would change the wording a little bit.

File: src/components/TheHeader.vue (hunk 1 above)
Comment (janmichek) [reply src/components/TheHeader.vue]: Agree, changed

**Review by michele-franchi — APPROVED** [review]
Minor typo but looks good

**Pull request merged**